### PR TITLE
docs: Fix multiple typos and grammar mistakes

### PR DESCRIPTION
**Pull request by Thor-x86**
Consider to use proper English for better perception among people.

### CODE REVIEW

**Comment by PippoRaimondi** [comment]
Thanks for your contribution.

File: README.md [hunk 1]
```diff
@@ -94,6 +94,6 @@ export WIDTH=1980 // Default is set to 720
 
 ### Please notice
 
-Be aware that despite forcing a screenshot resolution to a particular height and width for a test, if this test is run on different machinary i.e a 13inch mac vs computer attached to a 30inch monitor, the results will defer. So it's extremely important that you standerise where the tests will run, both locally and CI.
+Be aware that despite forcing a screenshot resolution to a particular height and width for a test, if this test is run on different computers (i.e a 13" Mac vs PC attached to a 30" monitor), the results will different. So it's extremely important that you standardize where the tests will run, both locally and CI.
```
Comment (PippoRaimondi) [inline README.md:97]: Maybe `the results will be different`?

Comment (Thor-x86) [reply README.md]: Fixed on ef6078b472870f69e8d715af1c5ce4d04ab4f336, please review

**Pull request merged**